charts/descheduler/templates/clusterrole.yaml

```diff
@@ -33,4 +33,9 @@ rules:
   resourceNames: ["{{ .Values.leaderElection.resourceName | default "descheduler" }}"]
   verbs: ["get", "patch", "delete"]
 {{- end }}
+{{- if .Values.metricsCollector.enabled }}
+- apiGroups: ["metrics.k8s.io"]
```

Comment:
can we have parity with this? https://github.com/kubernetes-sigs/descheduler/blob/master/kubernetes/base/rbac.yaml#L35-L37

```
- apiGroups: ["metrics.k8s.io"]
  resources: ["nodes", "pods"]
  verbs: ["get", "list"]
```